app/src/main/java/org/p2p/wallet/sdk/facade/SolendSdkFacade.kt

```diff
@@ -0,0 +1,211 @@
+            )
+        )
+            .getOrThrow()
+            .accounts
+    }
+
+    suspend fun getSolendConfig(): SolendConfigResponse = withContext(dispatchers.io) {
+        Timber.tag(TAG).i("Method getSolendConfig called")
+
+        methodResultMapper.fromSdk<SolendConfigResponse>(
+            solendSdk.getSolendConfig(
+                environment = solendEnvironment.sdkValue
+            )
+        )
+            .getOrThrow()
+    }
+
+    suspend fun getSolendDepositFees(
+        addressOwner: Base58String,
+        tokenAmountToDeposit: Long,
+        tokenAddressToDeposit: Base58String
+    ): SolendTokenDepositFees {
+        return withContext(dispatchers.io) {
+            Timber.tag(TAG).i("Method getSolendDepositFees called")
+
+            methodResultMapper.fromSdk<SolendTokenDepositFees>(
+                solendSdk.getSolendDepositFees(
+                    rpc_url = currentNetworkEnvironment.endpoint,
+                    owner = addressOwner.value,
+                    token_amount = tokenAmountToDeposit,
+                    token_symbol = tokenAddressToDeposit.value
+                )
+            )
+                .getOrThrow()
+        }
+    }
+
+    suspend fun getSolendMarketInfo(
+        tokens: List<Token>,
+        solendPoolName: String
+    ) {
+        return withContext(dispatchers.io) {
+            Timber.tag(TAG).i("Method getSolendMarketInfo called")
+
+            val tokenSymbols = tokens.joinToString(separator = ",", transform = Token::tokenSymbol)
+            methodResultMapper.fromSdk<SolendMarketInformation>(
+                solendSdk.getSolendMarketInfo(
+                    tokens = tokenSymbols,
+                    pool = solendPoolName
+                )
+            )
+                .getOrThrow()
+        }
+    }
+
+    suspend fun getSolendUserDepositByTokenSymbol(
+        userAddress: Base58String,
+        tokenSymbol: String,
+        solendPoolAddress: SolendPool
+    ): SolendUserDeposit = withContext(dispatchers.io) {
+        Timber.tag(TAG).i("Method getSolendUserDepositBySymbol with $tokenSymbol called")
+
+        methodResultMapper.fromSdk<SolendUserDepositByTokenResponse>(
+            solendSdk.getSolendUserDepositBySymbol(
+                owner = userAddress.value,
+                symbol = tokenSymbol,
+                pool = solendPoolAddress.poolAddress.value
+            )
+        )
+            .getOrThrow()
+            .userDepositBySymbol
+    }
+
+    suspend fun getAllSolendUserDeposits(
+        userAddress: Base58String,
+        solendPoolAddress: SolendPool
+    ): List<SolendUserDeposit> = withContext(dispatchers.io) {
+        Timber.tag(TAG).i("Method getSolendUserDeposits called")
+
+        methodResultMapper.fromSdk<SolendUserDepositsResponse>(
+            solendSdk.getSolendUserDeposits(
+                owner = userAddress.value,
+                pool = solendPoolAddress.poolAddress.value
+            )
+        )
+            .getOrThrow()
```

Comment:
this is redundant imho and the message is common for every call, that's not good as well